controllers/managedocs_controller.go

```diff
@@ -436,8 +433,15 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 			r.reconcileStrategy = v1.ReconcileStrategyNone
 		}
 
-		if err := r.get(r.addonParamSecret); err != nil {
-			return ctrl.Result{}, fmt.Errorf("Failed to get the addon param secret, Secret Name: %v", r.AddonParamSecretName)
+		// Read the secret and store it in addonParams map
```

Comment:
Can we be more specific with the comment? 
Maybe: // Read the add-on parameters secret and store it an addonParams map